### DISCUSSION TITLE
Alert/webhook when a stream goes live

### DISCUSSION THREAD

**Discussion by domainzero** [OvenMediaLabs/OvenMediaEngine#1486] [Ideas] [answered]
Is there any way currently to alert when a stream becomes active?
I was looking at the TranscodeWebhook but I'm not sure if that's exactly it.

I basically just want to get an alert/activate a webhook when a stream key goes live.

**Comment by bchah** [comment]
A bit hacky perhaps but you could configure the alert module for anything with a bitrate over 1kbps, i.e. any stream: https://airensoft.gitbook.io/ovenmediaengine/alert

**Comment by chris-ruecker** [comment, marked as answer]
You could use the admission webhook for this: https://airensoft.gitbook.io/ovenmediaengine/access-control/admission-webhooks

Just be aware it happens directly before the stream is live, so playback will not be ready in that instant.

**Reply by bchah** [reply]
This is the better answer!

**Reply by domainzero** [reply]
I ultimately got this to work by sending the payload to n8n, manipulating the data, and then sending it on.
Thanks for the tip!
